eng/common/scripts/update-docs-metadata.ps1

```diff
@@ -70,12 +70,14 @@ function GetAdjustedReadmeContent($pkgInfo, $lang){
     }
 
     $fileContent = $pkgInfo.ReadmeContent
-    $foundTitle = ""
```

Comment:
Looks like this is unrelated and might be a bad merge.

Reply:
Yeah. I have no intension of changing this file.